solvers/DPStokes/extra/poly_fits.h

```diff
@@ -0,0 +1,30 @@
+#include <vector>
+  
+namespace dpstokes_polys{
+
+  double polyEval(std::vector<double> polyCoeffs, double x){
+
+    int order = polyCoeffs.size() - 1;
+    double accumulator = polyCoeffs[order];
+    double current_x = x;
+    for(int i = 1; i <= order; i++){
+      accumulator += polyCoeffs[order-i]*current_x;
+      current_x *= x;
+    }
+
+    return accumulator;
+  }
+
```

Comment:
It would be great to have some comment here about what this is and where it comes from. Maybe a header in this file.

Reply:
Yeah I agree with this as well.

Reply:
Also for some reason it won't let me reply to your comments about the symmetry check - but that's a good point about the relative precision in the symmetry. Though like you said, it's probably best to do a mix of relative error (when the value is above e.g 1e-4) and absolute error (when the value is below that threshold)